source/stylesheets/refills/_search-tools.scss

```diff
@@ -66,6 +66,12 @@
         cursor: pointer;
         font-weight: 400;
         margin-bottom: 0;
+
+        &.summary {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4